leasing/management/commands/create_invoices.py

```diff
@@ -85,6 +85,9 @@ def handle(self, *args, **options):  # noqa: C901 'Command.handle' is too comple
                                 invoice_row_datum['invoice'] = invoice
                                 InvoiceRow.objects.create(**invoice_row_datum)
 
+                            # ensure 0€ total invoices get marked as PAID
+                            invoice.update_amounts()
```

Comment:
I would guess this makes the command run longer. As the update_amounts checks and queries all kinds of things.

As we are creating the invoice right here, we should already know if the invoice should be marked as paid.

I'd maybe just check the amount a few lines above and set the state as paid.

Reply:
Good point, will change.